ibis/tests/expr/test_table.py

```diff
@@ -2077,3 +2077,87 @@ def test_unbind_with_namespace():
 
     assert s.op() == expected.op()
     assert s.equals(expected)
+
+
+def test_table_bind():
+    def eq(left, right):
+        return all(a.equals(b) for a, b in zip(left, right))
+
+    t = ibis.table({"a": "int", "b": "string"}, name="t")
+
```

Comment:
would love a case for no args eg `t.bind()`. I would expect this to result in an empty tuple. Erroring would be OK if there is some problem with the empty tuple, but that would be my second choice.